tests/utils/cardContent.js

```diff
@@ -0,0 +1,26 @@
+
+const playgroundUrlPlaceholders = '${PLAYGROUND_PROXY}/${CURRENT_HOSTNAME}'
+
+const getCardAMPContent = async (browser, cardSelector) => {
+    const contentInfo = await browser.executeAsync(async (cardSelector, done) => {
+        let amp = document.querySelector(`${cardSelector} article`).firstElementChild.shadowRoot.AMP
+        let info = {url: amp.url, title: amp.title};
+        done(info)
+    }, cardSelector); 
+
+    return contentInfo
+}
+
+const hasRightContentLoaded = async (browser, cardSelector, contentConfig) => {
```

Comment:
💅🏽  To not confuse with left-right, what do you think about this name?

```suggestion
const isPageContentLoaded = async (browser, cardSelector, contentConfig) => {
```